src/Shutdown/ShutdownStrategyInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Bugsnag\Shutdown;
+
+use Bugsnag\Client;
+
+/**
+ * Interface ShutdownStrategyInterface.
+ */
+interface ShutdownStrategyInterface
+{
+    /**
+     * Register the shutdown behaviour.
+     *
+     * @param Client $client
+     */
```

Comment:
Should use a FQCN. Also, missing a return void annotations to let people know that they are not expected to return anything, Vs undocumented.

Reply:
Fixed, thanks.